diff --git a/node_modules/expo-share-intent/android/src/main/java/expo/modules/shareintent/ExpoShareIntentModule.kt b/node_modules/expo-share-intent/android/src/main/java/expo/modules/shareintent/ExpoShareIntentModule.kt
index ff5b100..f5e37f6 100644
--- a/node_modules/expo-share-intent/android/src/main/java/expo/modules/shareintent/ExpoShareIntentModule.kt
+++ b/node_modules/expo-share-intent/android/src/main/java/expo/modules/shareintent/ExpoShareIntentModule.kt
@@ -129,7 +129,7 @@ class ExpoShareIntentModule : Module() {
                         "text" to intent.getStringExtra(Intent.EXTRA_TEXT),
                         "type" to "text",
                         "meta" to mapOf(
-                            "title" to intent.getCharSequenceExtra(Intent.EXTRA_TITLE),
+                            "title" to (intent.getStringExtra(Intent.EXTRA_SUBJECT) ?: intent.getCharSequenceExtra(Intent.EXTRA_TITLE)),
                         )
                     ))
                 } else if (intent.action == Intent.ACTION_VIEW) {
